src/aat/resources/features/F-044/S-279.td.json

```diff
@@ -0,0 +1,23 @@
+{
+	"_guid_": "S-279",
+	"_extends_": "F-044_Test_Data_Base",
+
+	"title": "must return negative response when request does not provide valid authentication credentials",
+
+	"specs": [
+		"an active profile in CCD",
+		"does not provide valid authentication credentials",
+		"includes a HTTP 403 Forbidden"
+	],
+  "request": {
+    "headers": {
+      "Authorization": "eyJhbGciOiJIUzI1NiJ9"
```

Comment:
both this and s-280 are testing the same, shouldn't this scenario pass invalid user credentials?

Reply:
Scenario 279 is reserved for invalid credentials and prepared with wrong user Authorisation token but 280 is meant for unauthorised access and prepared with wrong ServiceAuthorization token instead of user authorisation.